.agents/workflows/cve-fix.md

```diff
@@ -0,0 +1,267 @@
+#### Fixing CVEs
+
+**Before starting, user must:**
+
+Install/update grype:
+```bash
+grype version || curl -sSfL https://get.anchore.io/grype | sudo sh -s -- -b /usr/local/bin
+```
+
+Fetch latest changes:
+```bash
+git fetch
+```
+
+##### 0. Update Vulnerability Database
+
+```bash
+grype db update
+```
+
+##### 1. Update Branch and Create Fix Branch
+
+```bash
+# Switch to target branch and update to latest
+git checkout release-0.X
+git merge --ff-only origin/release-0.X
+
+# Create fix branch (add -v2, -v3 etc if branch exists; don't delete old versions)
+git checkout -b fix-0.X-cves-YYYY-MM-DD
+```
+
+Replace `0.X`: `release-0.21` → `0.21`, `devel` → `devel`.
+
+One commit per package (may fix multiple CVEs), one PR total.
+
+##### 2. Scan
+
+```bash
+grype . --config .grype.yaml -o table
+```
+
+Ignore warning "no explicit name and version provided".
+
+For each CVE, note:
+- Package (**NAME**)
+- Version (**FIXED-IN**)
+- CVE ID (**VULNERABILITY**)
+
+**Note**: Same package may appear multiple times with different versions (e.g., quic-go v0.48.2 in coredns, v0.54.0 in main). Treat each as separate fix.
+
+##### 3. Locate Package
+
+```bash
+grep -Fl [package] go.mod coredns/go.mod tools/go.mod 2>/dev/null
+```
+
+**Check for replace directives**:
+```bash
+grep "replace.*$(basename [package])" go.mod coredns/go.mod tools/go.mod 2>/dev/null
+```
+
+If found, check git history why it was added. If obsolete, remove now. Otherwise update to safe version in Step 4:
+```bash
+# Check history
+git log -p --all -S "replace.*$(basename [package])" -- [module]/go.mod | head -50
+
+# Remove if obsolete
+sed -i '/replace.*[package]/d; /Fixes CVE-/d' [module]/go.mod
```

Comment:
Isn’t the “Fixes CVE-” deletion liable to delete more comments than we want?

`go mod edit -dropreplace` could be used for the first edit, but not to remove associated comments AFAIK.